src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        int number = random.nextInt(100);
+        DifColors difColors = colorSupplier.getRandomColor();
+        return ball;

```

Comment:
```suggestion
        return new Ball(number, difColors.value);
```
ти створюєш кульку але не ініціалізуєш її поля, для того щоб присвоїти їм значення створи конструктор з двома параметрами в класі Ball